components/org.apache.stratos.common/src/test/java/org/apache/stratos/common/test/ThriftClientConfigParserTest.java

```diff
@@ -44,48 +43,58 @@ public void testThriftClientConfigParser() throws Exception {
         URL configFileUrl = ThriftClientConfigParserTest.class.getResource("/thrift-client-config.xml");
         System.setProperty(ThriftClientConfig.THRIFT_CLIENT_CONFIG_FILE_PATH, configFileUrl.getPath());
         ThriftClientConfig thriftClientConfig = ThriftClientConfig.getInstance();
-        List <ThriftClientInfo> cepList = thriftClientConfig.getThriftClientInfo(
+        List<ThriftClientInfo> cepList = thriftClientConfig.getThriftClientInfo(
                 ThriftClientConfig.CEP_THRIFT_CLIENT_NAME);
-        List <ThriftClientInfo> dasList = thriftClientConfig.getThriftClientInfo(
+        List<ThriftClientInfo> dasList = thriftClientConfig.getThriftClientInfo(
                 ThriftClientConfig.DAS_THRIFT_CLIENT_NAME);
         ThriftClientInfo cepNode1 = null;
         ThriftClientInfo cepNode2 = null;
         ThriftClientInfo dasNode1 = null;
-        
+        ThriftClientInfo dasNode2 = null;
+
         for (ThriftClientInfo cepNodeInfo : cepList) {
-			if(cepNodeInfo.getId().equals("node-01")) {
-				cepNode1 = cepNodeInfo;
-			}else if(cepNodeInfo.getId().equals("node-02")) {
-				cepNode2 = cepNodeInfo;
-			}
-		}
-                
+            if (cepNodeInfo.getId().equals("node-01")) {
```

Comment:
better to use constant first approach when doing string comparison to avoid the NPE , ex 
if("node-01".equals(cepNodeInfo.getId()))